snippets/ztp-performance-profile.adoc

```diff
@@ -30,6 +30,11 @@ spec:
 <1> Ensure that the value for `name` matches that specified in the `spec.profile.data` field of `TunedPerformancePatch.yaml` and the `status.configuration.source.name` field of `validatorCRs/informDuValidator.yaml`.
 <2> Configures UEFI secure boot for the cluster host.
 <3> Set the isolated CPUs. Ensure all of the Hyper-Threading pairs match.
++
+[NOTE]
+====
+The reserved and isolated CPU pools must not overlap and together must span all available cores.
```

Comment:
Can we add extra bit of wording to be extra careful as leaving out any core causes an undefined behaviour? And add the "Important" tag like here https://docs.openshift.com/container-platform/4.10/scalability_and_performance/cnf-performance-addon-operator-for-low-latency-nodes.html#cnf-cpu-infra-container_cnf-master ?

Reply:
updated